changes/fix_GLT-3989.md

```diff
@@ -0,0 +1 @@
+version of ChromeDriver
```

Comment:
What do you think about `fix_GLT-3989.md`: `update version of ChromeDriver for tests`?

Reply:
Yep, that sounds more descriptive.